### PR TITLE
Add automatic install script for Linux

### PR DESCRIPTION
**Pull request by Virusmater**
Yes, I know it is not always secure, but it helps install on multiple machines.

If merged will be required to change README path from my repo to the main one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ The .exe file is an installer - just open it and install the native application.
 
 ### Installation on Linux
 
+Run script:
+
+`wget -O - https://raw.githubusercontent.com/Virusmater/firefox-webserial/master/install.sh | bash`
```
Comment (kuba2k2) [inline README.md:29]: Is is possible to use `curl` for this? It is more likely to have curl installed by default on a Linux distro than wget. Also curl is used anyway in the .sh script.

The URL has to be adjusted to point to the original repo :smile:

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (kuba2k2) [inline install.sh:1]: A better place for this file would be `native/install/linux_x86_64.sh`.

**Review by kuba2k2 — CHANGES_REQUESTED** [review]

File: install.sh (hunk 2 above)
Comment (Virusmater) [reply install.sh]: Done

File: README.md (hunk 1 above)
Comment (Virusmater) [reply README.md]: Done

**Review by kuba2k2 — APPROVED** [review]

**Pull request merged**